src/content/docs/en/reference/api-reference.mdx

```diff
@@ -2124,7 +2124,8 @@ console.log(foo + bar) // [!code focus]
 <Code
   code={code}
   lang="js"
-  transformers={[transformerNotationFocus()]} />
+  transformers={[transformerNotationFocus(),[transformerMetaHighlight()]}
+  meta="{1,3}" />
   
   <style is:global>
     pre.has-focused .line:not(.focused) {
```

Comment:
There is no style for `.line.highlighted` (see my original commit), so the code example doesn't really work (nothing happens when you add `meta`).